lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -100,13 +101,17 @@ def prune_whitelist
       wl + cluster_resource_discoverer.crds.select(&:prunable?).map(&:group_version_kind)
     end
 
+    def global_resources
+      cluster_resource_discoverer.globals.map { |g| g[:kind] }
+    end
+
     def server_version
       kubectl.server_version
     end
 
     def initialize(namespace:, context:, current_sha:, logger: nil, kubectl_instance: nil, bindings: {},
       max_watch_seconds: nil, selector: nil, template_paths: [], template_dir: nil, protected_namespaces: nil,
-      render_erb: true)
+      render_erb: true, allow_globals: false)
```

Comment:
Allows us to maintain backward compatibility of kubernetes-deploy. This will be removed when the exe files go away

Reply:
🤔Makes sense, but we should probably make sure that `Krane::DeployTask` never has this option. Maybe the transitional version we release will also need to have two classes in addition to two executables, where the `Krane` one is just delegating to `KubernetesDeploy`. Maybe we release the transitional version as `krane`, but as `0.1.0` and then `1.0.0` is when kubernetes-deploy is ripped out?

Reply:
I know rails used to release `X.99.0` right before `X+1.0.0` so you could bump to `X.99.0` and check for deprecation warnings before bumping to `X+1.0.0`. 

We're in a bit of branch from brach crunch. But, that could be a good pattern for making breaking changes.

Reply:
For backwards compatibility, shouldn't the default be true?